.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -107,6 +107,7 @@ body:
       description: How your application is managed? Not sure? Read [this part](https://docs.expo.dev/introduction/managed-vs-bare/) of Expo documentation
       options:
         - React Native (without Expo)
+        - React Native with Expo modules
         - Expo bare workflow
         - Expo managed workflow
```

According to @lukmccall it's outdated anyway. We can make it more actual:

```suggestion
      description: How your application is managed?
      options:
        - React Native 
        - Expo Go
        - Expo Dev Client
```